android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -734,6 +755,55 @@ private void finishPlayerInitialization() {
         applyModifiers();
     }
 
+    private void setupPlaybackService() {
+        if (showNotificationControls & player != null) {
+            playbackServiceConnection = new ServiceConnection() {
+                @Override
+                public void onServiceConnected(ComponentName name, IBinder service) {
+                    playbackServiceBinder = (PlaybackServiceBinder) service;
+
+                    try {
+                        playbackServiceBinder.getService().registerPlayer(player);
+                    } catch (Exception e) {
+                        DebugLog.e("ExoPlayer Playback", "Cloud not register ExoPlayer");
+                    }
+                }
+
+                @Override
+                public void onServiceDisconnected(ComponentName name) {
+                    playbackServiceBinder = null;
+                }
+
+                @Override
+                public void onNullBinding(ComponentName name) {
+                    DebugLog.e("ExoPlayer Playback", "Cloud not register ExoPlayer");
+                }
+            };
+
+            Intent intent = new Intent(themedReactContext, VideoPlaybackService.class);
+            intent.setAction(MediaSessionService.SERVICE_INTERFACE);
+
+            themedReactContext.startService(intent);
+
+            int flags;
+            if (Build.VERSION.SDK_INT >= 29) {
+                flags = Context.BIND_AUTO_CREATE | Context.BIND_INCLUDE_CAPABILITIES;
+            } else {
+                flags = Context.BIND_AUTO_CREATE;
+            }
+
+            themedReactContext.bindService(intent, playbackServiceConnection, flags);
+        }
+
+        if(!showNotificationControls && playbackServiceBinder != null) {
```

Comment:
That is strange to setupPlaybackService in a setup function ...
maybe you should split the function in 2 different functions ?